packages/backend/src/user/user.controller.ts

```diff
@@ -0,0 +1,44 @@
+import { Body, Controller, Post, Res } from '@nestjs/common';
+import { Response } from 'express';
+import {
+  DataSuccessResponse,
+  SimpleErrorResponse,
+  SimpleSuccessResponse,
+} from 'src/response.model';
+import { users as USERS } from 'src/test.data';
+
+@Controller('user')
+export class UserController {
+  @Post('/signin')
+  signin(
+    @Res({ passthrough: true }) res: Response,
+    @Body('name') name: string,
+    @Body('password') password: string,
+  ) {
+    const user = USERS.find(
+      (user) => user.name === name && user.password === password,
+    );
+    if (user) {
+      const token = Buffer.from(
+        JSON.stringify({ name: user.name, id: user.id }),
+      ).toString('base64');
+      return new DataSuccessResponse(token);
```

Comment:
the token is implemented as a base64 string, so anyone can fake other guy's token right?